docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,147 @@
+# Release 369 (19 Jan 2022)
+
+## General
+
+* Add support for `Pacific/Kanton` time zone. ({issue}`10679`)
+* Display `Physical input read time` using most succinct time unit in `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`10576`)
+* Fine tune request retry mechanism in HTTP event listener. ({issue}`10566`)
+* Add support for using PostgreSQL and Oracle with database backed resource
+  groups. ({issue}`9812`)
+* Remove unnecessary fine-grained spill toggles. ({issue}`10591`)
+* Remove distinction between system and user memory to simplify
+  cluster configuration. The configuration property
+  `query.max-total-memory-per-node` is removed. Use `query.max-memory-per-node`
+  instead. ({issue}`10574`)
+* Use formatting specified in the SQL standard when casting `double` and `real`
+  values to `varchar` type. ({issue}`552`)
+* Add experimental implementation of task level retries. ({issue}`9818`)
+* Prevent hanging query execution on failures with `phased` execution policy.
+  ({issue}`10656`)
+* Fix `UnsupportedOperationException` in `max_by` and `min_by` aggregation.
+  ({issue}`10599`)
+* Fix error message for function calls with lambda arguments. ({issue}`10423`)
+* Fix incorrect results or failure when casting date to bounded `varchar` type.
+  ({issue}`552`)
+
+## JDBC driver
+
+* Fix memory leak when using `DatabaseMetaData`. ({issue}`10584`,
+  {issue}`10632`)
+
+## BigQuery connector
+
+* Remove ``bigquery.case-insensitive-name-matching.cache-ttl`` configuration
+  option. It was previously ignored. ({issue}`10697`)
+* Fix query failure when reading columns with `numeric` or `bignumeric` type.
+  ({issue}`10564`)
+
+## ClickHouse connector
+
+* Upgrade minimum required version to 21.3. ({issue}`10703`)
+* Add support for [renaming schemas](/sql/alter-schema). ({issue}`10558`)
+* Add support for setting [column comments](/sql/comment). ({issue}`10641`)
+* Map ClickHouse `ipv4` and `ipv6` types to Trino `ipaddress` type.
+  ({issue}`7098`)
+* Allow mapping ClickHouse `fixedstring` or `string` as Trino `varchar` via the
+  `map_string_as_varchar` session property. ({issue}`10601`)
+* Disable `avg` pushdown on `decimal` types to avoid incorrect results.
+  ({issue}`10650`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Druid connector
+
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Hive connector
+
+* Add support for writing Bloom filters in ORC files. ({issue}`3939`)
+* Allow flushing cache for specific partition with the
+  [flush_metadata_cache](hive_flush_metadata_cache) system procedure.
+  ({issue}`10385`)
```

Comment:
It's not limited to specific partitions, it can be entire schemas, tables or partitions.

Maybe reword like `Allow flushing the metadata cache for specific schemas, tables or partitions with the...`?